xml/System.Text.Unicode/UnicodeRanges.xml

```diff
@@ -3172,8 +3180,8 @@
         <ReturnType>System.Text.Unicode.UnicodeRange</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the Yi Syllables Unicode block (U+A000-U+A48C).</summary>
```

Comment:
>A48C [](start = 63, length = 4)

A48F?

Reply:
I suggest you get the ranges from the file [https://raw.githubusercontent.com/dotnet/corefx/bffef76f6af208e2042a2f27bc081ee908bb390b/src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/UnicodeRanges.generated.cs](https://raw.githubusercontent.com/dotnet/corefx/bffef76f6af208e2042a2f27bc081ee908bb390b/src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/UnicodeRanges.generated.cs)

---
In reply to: [140909609](https://github.com/dotnet/docs/pull/3223#discussion_r140909609) [](ancestors = 140909609)